src/pages/workspaces/workspace/Data.js

```diff
@@ -81,13 +81,13 @@ export default hh(class WorkspaceData extends Component {
         }
       },
       Utils.cond(
-        [entitiesFailure, `Couldn't load workspace entities: ${entitiesFailure}`],
-        [!workspaceEntities, [spinner({ style: { margin: '2rem auto' } })]],
+        [entitiesFailure, () => `Couldn't load workspace entities: ${entitiesFailure}`],
+        [!workspaceEntities, () => [spinner({ style: { margin: '2rem auto' } })]],
```

Comment:
What is the difference between having the anonymous function return `[spinner({ style: { margin: '2rem auto' } })]` vs. `spinner({ style: { marginTop: '1rem' } })` (besides the margin, of course)?

Reply:
`children` needs to be an array if it's not a primitive. Because the default in this cond returns a multi-entry array, this needs to be its own array. The alternative would be wrapping the cond in an array and having the default return a Fragment.

Reply:
I see now. Thanks for the clarification.